webapp/lib/view.dart

```diff
@@ -459,38 +450,16 @@ class BatchRepliesConfigurationView extends PackageConfiguratorView {
     suggestedRepliesContainer = new DivElement()
       ..classes.add('conversation-responses');
     for (var suggestedResponse in data.suggestedReplies) {
-      suggestedRepliesContainer.append(
-        new DivElement()
-          ..classes.add('conversation-response')
-          ..append(
-            new ParagraphElement()
-              ..classes.add('conversation-response__language')
-              ..text = suggestedResponse['messages'][0]
-              ..contentEditable = 'true'
-          )
-          ..append(
-            new ParagraphElement()
-              ..classes.add('conversation-response__language')
-              ..text = suggestedResponse['messages'][1]
-              ..contentEditable = 'true'
-          )
-          ..append(
-            DivElement()
-              ..classes.add('conversation-response__reviewed')
-              ..append(
-                new CheckboxInputElement()
-                    ..classes.add('conversation-response__reviewed-state')
-                    ..checked = suggestedResponse['reviewed']
-              )
-              ..append(
-                new ParagraphElement()
-                ..classes.add('conversation-response__reviewed-description')
-                ..text = '${suggestedResponse['reviewed-by']}, ${suggestedResponse['reviewed-date']}'
-              )
-          )
-      );
+      suggestedRepliesContainer.append(_addSuggestedResponseEntry(suggestedResponse));
     }
 
+    suggestedRepliesContainer.append(
+      new ButtonElement()
+        ..classes.add('button-add-conversation-responses')
+        ..text = '+'
+        ..onClick.listen((event) => (event.target as Element).insertAdjacentElement('beforebegin', _addSuggestedResponseEntry()))
```

Comment:
between this and the add tag button, I think we can generalise. The only difference between them is what to do - perhaps we can pass that function as an argument? (the other current difference is the CSS class, but that can actually be the same)

Reply:
I'm not totally understanding this

Reply:
it might be easier to show what I mean than explain it in text... I've opened an issue for me to not forget about this. It's a low importance refactoring, so I'll merge this PR without this in